linkerd/app/integration/src/controller.rs

```diff
@@ -232,7 +228,7 @@ impl pb::destination_server::Destination for Controller {
         let _e = span.enter();
         tracing::debug!(request = ?req.get_ref(), "received");
 
-        if let Ok(mut calls) = self.expect_dst_calls.lock() {
+        let mut calls = self.expect_dst_calls.lock();
             if self.unordered {
```

Comment:
i believe this formatting change is why rustfmt is failing :) since the lock is no longer an `if let`, the rest of the code shouldn't be indented any longer.